src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -620,43 +620,20 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         }
 
         // Create table body
-        StringBuilder tableBodyHtml = new StringBuilder();
-
-        String tableBodyFragmentTemplate = FormTemplates.RUBRIC_RESULT_STATS_BODY_FRAGMENT;
-        String tableBodyTemplate = FormTemplates.RUBRIC_RESULT_STATS_BODY;
-        DecimalFormat df = new DecimalFormat("#");
-        DecimalFormat dfAverage = new DecimalFormat("0.00");
+        StringBuilder tableBodyHtml;
```

Comment:
It's usually a bad practice to define a varailbe far away from where it is first used.

Although we don't really enforce this http://checkstyle.sourceforge.net/config_coding.html#VariableDeclarationUsageDistance